tutorials/ab3AddRemark.md

```diff
@@ -10,7 +10,11 @@ Let's walk you through the implementation of a new command — `remark`.
 
 This command allows users of the AddressBook application to add optional remarks to people in their address book and edit it if required. The command should have the following format:
 
-`remark INDEX r/REMARK` (e.g., `remark 2 r/Likes baseball`)
+`remark INDEX r/REMARK`
+* `INDEX` refers to the list position of the person you want to add a remark to.
+* `REMARK` is the note you wish to attach to their entry.
+
+For example, `remark 2 r/Likes baseball` adds the remark “Likes baseball” to the current person at index 2 of the observable list.
```

Comment:
Similarly, could consider making a slight modification to use a more general term so that it is more reader-friendly / less technical rather than "observable":

* For example, `remark 2 r/Likes baseball` adds the remark “Likes baseball” to the person at index 2 of the currently displayed person list.